README.md

```diff
@@ -17,17 +17,17 @@ Appwrite is an open-source backend as a service server that abstract and simplif
 Add this reference to your project's `.csproj` file:
 
 ```xml
-<PackageReference Include="Appwrite" Version="0.5.0" />
+<PackageReference Include="Appwrite" Version="0.6.0" />
 ```
 
 You can install packages from the command line:
 
 ```powershell
 # Package Manager
-Install-Package Appwrite -Version 0.5.0
+Install-Package Appwrite -Version 0.6.0
```

Comment:
I only did a minor version bump here since moving to 1.0.0 would indicate we're going stable with dot net which we are not